docs/layouts/shortcodes/endpoint-description.md

```diff
@@ -130,6 +130,7 @@ mounts:
 
 <!-- Don't add use in app intro to Headless Chrome, which has different content -->
 {{ if ne ($type) "chrome-headless" }}
-Then use the service in your app with a configuration file like the following:
+To use the configured service in your app,
+add a configuration file similar to the following in your project:
```

Comment:
:+1: I think this is clearer that the original. Had read it so often, just overlooked the ways it could be improved.

:hammer_and_wrench: 
```suggestion
To use the configured service in your app,
add a configuration file similar to the following to your project:
```